addons/project/controllers/rating.py

```diff
@@ -32,14 +32,14 @@ def _calculate_period_partner_stats(self, project_id):
                 rating_rating
             WHERE
                 parent_res_model = 'project.project'
-                    AND parent_res_id = %s
+                    AND parent_res_id = %%s
                     AND res_model = 'project.task'
                     AND rated_partner_id IS NOT NULL
                     AND create_date >= current_date - interval '90' day
                     AND rating IN (1,5,10)
             GROUP BY
                 rated_partner_id, rating, period
-        """ % (project_id))
+        """, (project_id))
```

Comment:
There is a comma missing: `(project_id)` is not a tuple!

Reply:
@rco-odoo Changes are done as per suggestion.
Thanks,